fdbserver/storageserver.actor.cpp

```diff
@@ -6798,8 +6800,10 @@ ACTOR Future<Void> fetchKeys(StorageServer* data, AddingShard* shard) {
 		// We must also ensure we have fetched all change feed metadata BEFORE changing the phase to fetching to ensure
 		// change feed mutations get applied correctly
 		state std::vector<Key> changeFeedsToFetch;
-		std::vector<Key> _cfToFetch = wait(fetchCFMetadata);
-		changeFeedsToFetch = _cfToFetch;
+		if (!isFullRestoreMode()) {
```

Comment:
Why do we want to skip feeds fetching during restore?
In the initial phase from the blob migrator there will be no feeds to fetch, but if we're fetching from another storage instead later, I think we need to fetch its feeds. The better solution might be to have the blob migrator return an empty set of feeds to fetch instead for getOverlappingChangeFeeds?

Reply:
Yes.. you are right. StorageServer::fetchChangeFeedMetadata() is stuck during restore. During restore, there is no new writes from tlog servers so it keeps waiting for a version(see StorageServer::overlappingChangeFeedsQ()). 

I'll need the input from you and @etschannen. Currently I simply ignore it.. if you both think we should return empty feeds in this case, it also works for me.

Reply:
This is fine for now, because after a restore is complete we should forget all the history and re-enable hybrid from scratch. When we want to use this for bulk loading data we will need a better solution.

Reply:
well noted. Thank you both for comments!